lib/source_file.cpp

```diff
@@ -242,13 +265,29 @@ cstring SourceInfo::toSourceFragment() const {
     return InputSources::instance->getSourceFragment(*this);
 }
 
+cstring SourceInfo::toSourceFragment2() const {
+    return InputSources::instance->getSourceFragment2(*this);
+}
+
 cstring SourceInfo::toPositionString() const {
     if (!this->isValid())
         return "";
     SourceFileLine position = InputSources::instance->getSourceLine(this->start.getLineNumber());
     return position.toString();
 }
 
+cstring SourceInfo::toSourcePositionData(unsigned *outLineNumber,
+                                         unsigned *outColumnNumber) const {
+    SourceFileLine position = InputSources::instance->getSourceLine(this->start.getLineNumber());
```

Comment:
Understood.  My reason for writing it that way is that I copied, pasted, and edited this method from toPositionString() above it, and it seems to be working as desired.